WootricSDK/WootricSDK/UIItems.m

```diff
@@ -124,27 +143,37 @@ + (UILabel *)customThankYouLabelWithFont:(UIFont *)font {
 }
 
 + (UIButton *)facebookButtonWithTargetViewController:(UIViewController *)viewController {
-  UIButton *tempButton = [[UIButton alloc] init];
-  tempButton.hidden = YES;
-  [tempButton setImage:[UIImage imageNamed:@"facebook_icon"] forState:UIControlStateNormal];
-  [tempButton setTranslatesAutoresizingMaskIntoConstraints:NO];
-  [tempButton addTarget:viewController
-                 action:NSSelectorFromString(@"facebookButtonPressed")
-       forControlEvents:UIControlEventTouchUpInside];
-
-  return tempButton;
+    [self dinamycallyAddFont];
+    
+    UIButton *tempButton = [UIButton buttonWithType:UIButtonTypeRoundedRect];
+    tempButton.hidden = YES;
+    
+    [tempButton.titleLabel setFont:[UIFont fontWithName:kFontAwesomeFamilyName size:20]];
+    [tempButton setTitle:[NSString fontAwesomeIconStringForEnum:FAFacebook] forState:UIControlStateNormal];
+    [tempButton.titleLabel setTextColor:[WTRColor facebookLogoTextColor]];
+    [tempButton setTranslatesAutoresizingMaskIntoConstraints:NO];
+    [tempButton addTarget:viewController
+                   action:NSSelectorFromString(@"facebookButtonPressed")
+         forControlEvents:UIControlEventTouchUpInside];
```

Comment:
@diablourbano like your suggestion. I'll refactor it to a generic method. Besides I have to add another button, this will make it simpler.